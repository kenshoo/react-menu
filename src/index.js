export { Menu } from "./components/menu";
