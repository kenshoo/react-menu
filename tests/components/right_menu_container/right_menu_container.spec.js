import React from "react";
import renderer from "react-test-renderer";

import { RightMenuContainer } from "../../../src/";

describe("RightMenuContainer component", () => {
  test("renders children", () => {
    const cmp = renderer.create(<RightMenuContainer>TEST</RightMenuContainer>);
    expect(cmp).toMatchSnapshot();
  });

  test("recieves className", () => {
    const cmp = renderer.create(
      <RightMenuContainer className="TEST_CLASS">TEST</RightMenuContainer>
    );
    expect(cmp).toMatchSnapshot();
  });
});
