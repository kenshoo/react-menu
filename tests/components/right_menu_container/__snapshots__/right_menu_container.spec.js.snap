// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightMenuContainer component recieves className 1`] = `
<div
  className="right_menu_container TEST_CLASS"
>
  TEST
</div>
`;

exports[`RightMenuContainer component renders children 1`] = `
<div
  className="right_menu_container"
>
  TEST
</div>
`;
